### PR TITLE
Dockerfile: use alpine 3.12

### PR DESCRIPTION
**Pull request by aanm**
In Alpine 3.11 and 3.13, 'nslookup' exits with the error code 1 if it
can't resolve all IPs for the search list defined in /etc/resolv.conf

However, it seems that Alpine 3.10 and 3.12 are not affected by this bug
and continue return the error code 0 if at least on of the domains in
the search list is resolved into an IP address. Thus, we will use the
latest Alpine image available for the 3.12 release series.

Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

File: Documentation/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/python:3.7.9-alpine3.11
+FROM docker.io/library/python:3.7.10-alpine3.12@sha256:ffe14871002b68eb7ca9583d7b0bf991d9066f7d9f9cb713c6510e966750ef19
```
Comment (sayboras) [inline Documentation/Dockerfile]: I know that alpine 3.13 is having some issues as I got one big incident related to dns recently, but didn't know 3.11 is also problematic. Can you share some details if I might ? Thanks.

Comment (aanm) [reply Documentation/Dockerfile]: Simply deploy this snippet and you will see that alpine 11 and 13 will fail:
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    k8s-app: alpine-checker
  name: alpine-checker
  namespace: default
spec:
  replicas: 1
  selector:
    matchLabels:
      k8s-app: alpine-checker
  template:
    metadata:
      labels:
        k8s-app: alpine-checker
      name: alpine-checker
    spec:
      containers:
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.10
        imagePullPolicy: IfNotPresent
        name: alpine-3-10
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.11
        imagePullPolicy: IfNotPresent
        name: alpine-3-11
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.12
        imagePullPolicy: IfNotPresent
        name: alpine-3-12
      - command:
        - sh
        args:
        - -c
        - 'if ! nslookup kubernetes &>/dev/null; then echo fail; else echo success && sleep 99999d; fi'
        image: docker.io/library/alpine:3.13
        imagePullPolicy: IfNotPresent
        name: alpine-3-13
      dnsPolicy: ClusterFirst
      restartPolicy: Always
```

**Comment by rolinh** [comment]
@aanm See my comment in [this PR](https://github.com/cilium/hubble/pull/540#pullrequestreview-649022687). TLDR: it seems broken also in 3.12, the only difference being that it exits with code 0 instead of 1 so appears successful.  Have you tried `bind-tools`'s implementation of `nslookup` instead?

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-gke

**Comment by aanm** [comment]
ci-multicluster

File: Documentation/Dockerfile (hunk 1 above)
Comment (sayboras) [reply Documentation/Dockerfile]: thanks :100: , the above is nice :100:

**Comment by aanm** [comment]
test-gke

**Review by rolinh — APPROVED** [review]

**Review by qmonnet — APPROVED** [review]

File: install/kubernetes/cilium/Chart.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 1.9.90
-appVersion: 1.9.90
+version: 1.10.90
```
Comment (nebril) [inline install/kubernetes/cilium/Chart.yaml:5]: shouldn't this be `1.11.90` for master?

**Review by nebril — APPROVED** [review]
LGTM overall. Would be nice to have all these shas defined in one place only, but I guess it's out of scope of this change.

One question left inline, but non-blocking.

File: install/kubernetes/cilium/Chart.yaml (hunk 2 above)
Comment (aanm) [reply install/kubernetes/cilium/Chart.yaml:5]: No, it's 1.10.90 as we haven't released 1.11 yet. See it as the `90` is between `1.10.x` and `1.11.0`

**Review by borkmann — APPROVED** [review]

**Pull request merged**